backend/src/API/teamEventsAPI.ts

```diff
@@ -72,8 +72,7 @@ export const requestTeamEventCredit = async (
       `User with email ${user.email} cannot request team event credit for another member, ${request.member.email}.`
     );
   }
-  const updatedteamEvent = { ...request, pending: true };
-  await teamEventAttendanceDao.createTeamEventAttendance(updatedteamEvent);
+  await teamEventAttendanceDao.createTeamEventAttendance(request);
```

Comment:
What's the purpose of this change?

Reply:
The team event request from the frontend should already have pending set to true by default, so this would be redundant.

Reply:
Backend shouldn't assume that data from the frontend is "correct"

Reply:
👍🏽